packages.yml

```diff
@@ -3,4 +3,10 @@ packages:
     version: 0.1.9
 
   - package: fishtown-analytics/snowplow
-    version: 0.6.2    
\ No newline at end of file
+    version: 0.6.2
+
+  - package: fishtown-analytics/adwords
```

Comment:
i tagged you in the PR there should be a new version of adwords out hopefully tomorrow